src/features/dialog-manager/dialogs/inventory-dialog/index.js

```diff
@@ -1,18 +1,25 @@
 import React, { useState } from 'react';
+import { connect } from 'react-redux';
 
 import Backpack from './backpack.png';
 import BackpackItems from '../../../../components/backpack-items';
 import Dialog from '../../../../components/dialog';
 import EquippedItems from '../../../../components/equipped-items';
 import ViewItem from '../../../../components/view-item';
+import toggleInventory from '../../actions/toggle-inventory';
 
 import './styles.scss';
 
-const InventoryDialog = () => {
+const InventoryDialog = ({ toggleInventory }) => {
     const [viewItem, setViewItem] = useState(false);
 
     return (
-        <Dialog>
+        <Dialog
+            keys={[73]}
```

Comment:
Is it worth making a macro to remove the magic numbers? (Could also do it with the spade key and any other keys used if you want

Reply:
I've created an issue to address this #110